### PR TITLE
EHUB-110 : change environment variable name

### PR DESCRIPTION
**Pull request by AkhileshPrajapatiNV**
change environment variable name HUB_GROUP_DETAIL_BASE_URL

### CODE REVIEW

**Review by SACHIN-BAGHEL — APPROVED** [review]

**Review by SACHIN-BAGHEL — COMMENTED** [review]
Changed the environment variables to capital as per comment in https://entando.myjetbrains.com/youtrack/issue/EHUB-110

File: application/src/main/resources/config/application-dev.yaml [hunk 1]
```diff
@@ -49,5 +49,4 @@ keycloak:
   resource: ${keycloak.client-id}
   use-resource-role-mappings: true
 
-app:
-  hub-group-detail-base-url: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
+HUB_GROUP_DETAIL_BASE_URL: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
```
Comment (nshaw) [inline application/src/main/resources/config/application-dev.yaml:52]: @SACHIN-BAGHEL this is mixing the local stack settings with the integration environment setting.

Comment (nshaw) [reply application/src/main/resources/config/application-dev.yaml:52]: This one should probably be changed to http://localhost:3000/#/ so it's setup for a local environment,

Comment (SACHIN-BAGHEL) [reply application/src/main/resources/config/application-dev.yaml:52]: I Will make that change.

**application.yaml** - HUB_GROUP_DETAIL_BASE_URL: http://hubdev.okd-entando.org/entando-de-app/en/test.page#/
**application-dev.yaml** - HUB_GROUP_DETAIL_BASE_URL: http://localhost:3000/#/
**application-prod.yaml** - ?
 
**What is the expected value in application-prod.yaml ?**

Comment (nshaw) [reply application/src/main/resources/config/application-dev.yaml:52]: @SACHIN-BAGHEL , let's leave it commented out and empty in application.yaml and application-prod.yaml. Please also add a comment in the README about setting the value if you want active links back from the App Builder.

**Pull request merged**